src/body.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import { getNasaImg } from './nasapic'
+import { StyledImg } from './StyledImg'
+import { StyledBody } from './StyledBody'
+
+const Body = () => {
+    const [img, setImg] = useState();
```

Comment:
personal choice again, but I think it would be easier and also more OOP if you just had 1 dataObj state field and then just access what you need using dot/bracket notation.